plugin/pulpcore/plugin/download/factory.py

```diff
@@ -106,8 +108,9 @@ def _https(self, url, path=None, artifact=None):
         Returns:
             HttpDownload: An https download.
         """
-        if artifact:
-            _path = artifact.relative_path
+        if remote_artifact:
+            import os
+            _path = os.path.join("/var/lib/pulp/tmp", remote_artifact.sha256)
```

Comment:
Why are we doing this?  We should not be hard coding paths.  the downloader should be configured to download into the working_dir which is the CWD.